deployments/README.md

```diff
@@ -0,0 +1,72 @@
+# Nanocloud deployments
+
+This directory is a collection of tools to deploy Nanocloud.
+
+## Install with ansible
+
+### Dependencies
+
+You need ansible 1.9 or above to use this installation
+
+Install dependencies
+
+```
+ansible-galaxy install franklinkim.docker
+ansible-galaxy install franklinkim.docker-compose
+```
+
+> Note: You may need to have root privilege to install those dependencies. You
+> won't need it to run deployment.
+
+### Configure
+
+Configure where *ansible* will deploy your installation by modifying
```

Comment:
Either:
by modifying **the** ansible_hosts file.
or
by modifying ansible_hosts